### ISSUE TITLE
two questions for SAN

### ISSUE THREAD

**Issue by QcQcM** [D-X-Y/landmark-detection#59]
Hello, I have two questions.
First, in cluster.py, does finetune resnet101 only train for two rounds? What is the classification accuracy of finetuned's network for four styles? (because my computer crashed when running this part of code, I don't know the actual accuracy,  I used the 300W style convert data set to train resnet50 for two rounds and the classification accuracy is very poor.)2. Why choose gray, light, sketch for style aggregation? Why not style else?
Hope to get your reply, thank you very much

**Comment by D-X-Y** [comment]
Yes, only finetune for two epochs.
Please see readme here for the accuracy: https://github.com/D-X-Y/landmark-detection/tree/master/SAN#datasets-download . In short, over 90%.

We choose 4 representative styles in the paper, the algorithm is applicable to all different styles.
